### ISSUE TITLE
Apple API usage description needed

### ISSUE THREAD

**Issue by Sumbersss** [tripflex/WifiWizard2#81]
Check all boxes if you have done the following:
- [x] Checked that your issue isn't already filed: https://github.com/tripflex/wifiwizard2/issues
- [x] Make sure you fill out the Issue Type below

### Issue type
Select all that apply
- [ ] Bug
- [x] Enhancement
- [ ] Task
- [ ] Question
- [ ] Other

### Description

Since spring 2019 apple now need explanation on user sensitive datas usage so  when I submit my app I now have this issues which are from the WifiWizard2 plugin

> Missing Purpose String in Info.plist File - Your app's code references one or more APIs that access sensitive user data. The app's Info.plist file should contain a NSLocationAlwaysUsageDescription key with a user-facing purpose string explaining clearly and completely why your app needs the data. Starting Spring 2019, all apps submitted to the App Store that access user data will be required to include a purpose string. If you're using external libraries or SDKs, they may reference APIs that require a purpose string. While your app might not use these APIs, a purpose string is still required. You can contact the developer of the library or SDK and request they release a version of their code that doesn't contain the APIs. Learn more (https://developer.apple.com/documentation/uikit/core_app/protecting_the_user_s_privacy).

Missing Purpose String in Info.plist File - Your app's code references one or more APIs that access sensitive user data. The app's Info.plist file should contain a NSLocationWhenInUseUsageDescription key with a user-facing purpose string explaining clearly and completely why your app needs the data. Starting Spring 2019, all apps submitted to the App Store that access user data will be required to include a purpose string. If you're using external libraries or SDKs, they may reference APIs that require a purpose string. While your app might not use these APIs, a purpose string is still required. You can contact the developer of the library or SDK and request they release a version of their code that doesn't contain the APIs. Learn more (https://developer.apple.com/documentation/uikit/core_app/protecting_the_user_s_privacy).

### Steps to Reproduce

1. archive an app with the Wifiwizard2 plugin

**Expected behavior:** no warning form Apple Store team

**Actual behavior:** I received warning for Apple Store team telling me to describe API usage

**Reproduces how often:** 100%

### Versions

Wifiwizard2 3.1.0

### Additional Information

**Comment by pierresh** [comment]
Yes, that is a bit annoying. It would be great we could deactivate this. I suppose we do not need "always" accessing user's location.

**Comment by tripflex** [comment]
@pierresh @Sumbersss so is this specific to accessing the user's location?  Guess i'm just trying to understand what APIs the issue is referring to ... and how we would add `NSLocationAlwaysUsageDescription` and `NSLocationWhenInUseUsageDescription` to this lib

**Comment by tripflex** [comment]
Found this:
https://stackoverflow.com/questions/33637591/setting-nslocationwheninuseusagedescription-value-when-using-cordova-plugin-geol

We can probably use the same handling as this lib to do this:
https://github.com/apache/cordova-plugin-geolocation/commit/f9f7a23c57daad5514ce3d830a2defeb93ba3f78

Not sure whether or not we should add a static defined string, or if we should have it dynamically generated like the lib above does.  Thoughts?
